### ISSUE TITLE
[VSCode] Rendering Deletes ipynb File

### ISSUE THREAD

**Issue by Servinjesus1** [quarto-dev/quarto#113]
When I render a qmd in a directory with a notebook, it overwrites and deletes the notebook. This happens both when rendering on the CLI and when using "Quarto: Render Project".

I believe this is happening during execution. Can Quarto please rely on a temporary notebook for rendering qmd files rather than potentially clobbering the existing one?

Quarto-VS v1.73.0

**Comment by jjallaire** [comment]
I agree we should use a more fitting temporary file name here! We can't do this for v1.3 because we are just about to ship, but we will fix this very early in the v1.4 development cycle. Transferring this to quarto-cli for work during v1.4.
